www/src/pages/components/image/react/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Image
+description: Commonly used image styles.
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader, ComponentFooter } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+## Examples
+
+### Magic Image
+
+```jsx
+<MagicImage id="350357374752325652" aspectRatio={864 / 1600} />
```

Comment:
Not sure how to handle this but there are two scenarios for `aspectRatio`:

1. The consumer wants an image in one of our available [aspect ratios](https://thumbprint.design/guide/product/aspect-ratio/). 
2. The consumer wants an image, isn't requesting a specific aspect ratio, but wants to hold the space.

They are different concepts though not sure if it's worth trying to decouple them. I can imagine a consumer requesting an unofficial aspect ratio, like 5/6, and wondering why the image service is not returning an image in that aspect ratio (it would only return the original aspect ratio).